scripts/rpm-setup-autosign

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash -e
+
+rpmhome=$(rpm --eval "%{xdg:config}/rpm")
+macros="macros"
+# if oldstyle config exists, then use that instead
+if [[ ! -d "${rpmhome}" && ( -f ~/.rpmmacros || -f ~/.rpmrc) ]]; then
+    rpmhome="${HOME}"
+    macros=".rpmmacros"
+fi
+mkdir -p "${rpmhome}"
+
+email="rpmbuild-${USER}@$(uname -n)"
+keypath="${rpmhome}/${email}.asc"
+
+function log()
+{
+    echo $* 1>&2
+}
+
+function progpath()
+{
+    which ${1} 2> /dev/null
+}
+
+function checkprog()
```

Comment:
This function isn't used (I suspect it previously was and just got refactored away). Maybe we could still use it to check if the gpg/sq program exists but that might also be overly pedantic...

Reply:
Yeah its another leftover from various experiments. I'll just kick it.